tests/cli/core/products/services/test_product_service.py

```
@@ -1,3 +1,4 @@
+# noqa: WPS204
```

Ruff (0.15.1)

[warning] Unused `noqa` directive (unknown: `WPS204`)

Remove unused `noqa` directive

(RUF100)